tools/golang_patched/README.md

```diff
@@ -0,0 +1,26 @@
+# Patched Golang Docker image
```

Comment:
My only comment would be to mention DCE in this title and change the directory to "golang_patched_dce" to make this easier to find. Otherwise, super great detective work. Thank you!